lib/Less/Functions.php

```diff
@@ -123,7 +123,7 @@ public function hsva( $h, $s, $v, $a ) {
 		$v = Less_Functions::number( $v );
 		$a = Less_Functions::number( $a );
 
-		$i = floor( ( $h / 60 ) % 6 );
+		$i = floor( (int) ( $h / 60 ) % 6 );
```

Comment:
I don't follow the reaosn for this rounding - is the expression ($h / 60) somehow not resulting in an int or float?

Reply:
Yea, ($h / 60) results in a float and modulus `%` requires an `int`